packages/extension/src/modules/twitch.ts

```diff
@@ -57,24 +57,27 @@ async function injectChat (element: HTMLElement) {
     const badgesContainer = element.parentElement?.parentElement?.parentElement?.firstChild
     if (!badgesContainer) return
 
+    const pronounBadge = h(
```

Comment:
Why move this as a variable? I guess it can improve code readability a but I don't see an issue with having it inlined

Reply:
I was finding having the many nested parts hard to understand which part was created element. I had moved it into a variable cause I assumed I'd have do something like:

```js
pronounBadge.classList.add('pronoundb-pronoun-badge');
```

Can change this back if you want